src/pe_reports/pages.py

```diff
@@ -18,6 +18,37 @@
 class Pages:
     """Class containing the presentation page information."""
 
+    @staticmethod
+    def add_overview_value(run, df):
+        """Add summary stats to the overview page."""
+        run.text = str(int(df.iloc[0]["count"]))
+        font = run.font
+        Paragraph.text_style_ov_val(font)
+
+    @staticmethod
+    def insert_chart(slide, df_loc, col_names, chart_type, size, x, y, cx, cy, stacked):
+        """Insert charts into the PowerPoint."""
+        chart = CategoryChartData()
+        try:
+            # TODO: Remove hard-coded file locations
```

Comment:
Probably could remove the TODO here:
            # TODO: Remove hard-coded file locations
            # Issue https://github.com/cisagov/pe-reports/issues/8
            
But we want to add back in where there are file paths in the page builds for reference.  If they were originally missing, might as add in now.